holonote/tests/conftest.py

```diff
@@ -111,3 +92,16 @@ def multiple_fields_annotator(conn_sqlite_uuid) -> Annotator:
         connector=conn_sqlite_uuid,
     )
     return anno
+
+
+@pytest.fixture()
+def element_range1d() -> hv.Image:
```

Comment:
Looks like the type is declared wrong here.

Reply:
Yep, a clear case of ctrl+c and ctrl+v

```suggestion
def element_range1d() -> hv.Curve:
```